src/core/unit.c

```diff
@@ -6349,7 +6349,7 @@ void unit_freezer_complete(Unit *u, FreezerState kernel_state) {
                                          freezer_state_to_string(u->freezer_state));
 
         /* If the cgroup's final state is against what's requested by us, report as canceled. */
-        bus_unit_send_pending_freezer_message(u, /* canceled = */ !expected);
+        bus_unit_send_pending_freezer_message(u, /* cancelled = */ !expected);
```

Comment:
I prefer the other spelling ;-) but ok

Reply:
Ah, this is UK'ism versus US'ism. I suppose systemd prefers the US version so let me just fix that

Reply:
Since its not that much work, I've fixed up bus_unit_send_pending_freezer_message